fe/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -188,6 +199,15 @@ public void write(DataOutput out) throws IOException {
         for (long errorReplciaId : errorReplicas) {
             out.writeLong(errorReplciaId);
         }
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_46) {
```

Comment:
em...now, this is work in progress, you'd better not to persist any object. Because code would be changed later. If you persist something, there will be more work to do compatible.